src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx

```diff
@@ -0,0 +1,192 @@
+
+import apmTraceDetails from 'images/apm_screenshot-crop_trace-details.png'
+
+import apmTransactionTracesinAPM from 'images/apm_screenshot-full_transaction-traces-in-APM.gif'
+
+import apmTransactionsTabinApmOverview from 'images/apm_screenshot-full_transactions-tab-in-apm-overview.gif'
+
+
+We all know what it's like to wake up in the morning to a ping from your leadership team telling you that something is going wrong and they need it fixed as soon as possible. This is where our APM tools come in. When you have limited time and increased urgency, you can use APM to run diagnostics and triage issues in eight clicks. 
+
+For this example, we're going to explore a situation where you work for an e-commerce company and you manage the checkout team. You are alerted to the fact that the `Pay Now` button is failing and customers can't checkout. 
+
+<Steps>
+
+<Step>
+### Open APM [#open-apm]
+Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > All capabilities > APM and Services**.
+
+From there, click on the application that's causing the `Pay Now` button to fail. For this example, the app will be called `Checkout-service`.
+
+<img
+    title="APM summary page for checkout service"
+    alt="A screenshot showing the APM summary page"
+    src={apmCheckoutServiceinApm}
+/>
+</Step>
+
+<Step>
+
+### Select a time period [#time-picker]
+
+It's important to make sure that you are looking at the right time segment. Based on the message you got from your leadership team, you know the checkout issues are happening now and so you want to hone in on what's actively happening in your system. Click on the **Time Picker** in the top-left, and select **Since 30 minutes ago**.
+
+    <img
+    title="Time picker in APM"
+    alt="A screenshot that demonstrates how to select the time period for APM"
+    src={apmTimePickerinApm}
+/>
+</Step>
+
+<Step>
+
+### Find the problematic transaction [#transactions]
+
+You know that checkout is failing in your app but you're unsure of what's causing the actual error. In order to figure out what part of the checkout process is breaking you need to investigate your apps `Transactions`. A transaction represents the “work” that happens from when your application receives a request to when it sends a response. In order to determine which transaction is breaking, click on the **Transactions** tab.
+
+A slow transaction can be a strong indicator that something is behaving abnormally, so digging deeper into your slowest transaction is a good place to start. Once you click into **Transactions** you will see a list of all the "work" that is happening behind the scenes for your `checkout-service` application. On the left-hand side you will see which transaction is taking the most time. For the purpose of this example, the slowest transaction is `Sinatra/Rubytron/POST #purchaseCart.` Click on the transaction.
```

Comment:
I think that stuff like "For the purpose of this example" should actually be standardized. I don't think there's a right one to use, but it should be the same in all accounts.

Reply:
Good call!